web/styles/components.css

```diff
@@ -96,7 +96,13 @@ a.no-underline:hover {
 i.zulip-icon.zulip-icon-bot {
     color: var(--color-icon-bot);
     vertical-align: top;
-    padding: 0 2px;
+    padding: 0 0 0 5px;
+}
+
+.pill-label {
+    & i.zulip-icon.zulip-icon-bot {
```

Comment:
If it's possible, simplify this nested selector to just `.zulip-icon-bot`

Reply:
updated...